src/cards/debilitate.ts

```diff
@@ -41,14 +41,16 @@ const spell: Spell = {
   },
   events: {
     onTakeDamage: (unit, amount, _underworld, damageDealer) => {
-      const quantity = unit.modifiers[id]?.quantity || 1;
-      // Magnify positive damage
-      if (amount > 0) {
-        return amount * proportion * quantity;
-      } else {
-        // Do not magnify negative damage (which is healing)
-        return amount;
+      const modifier = unit.modifiers[id];
+      if (modifier) {
+        // Will only increase damage (doesn't affect incoming healing)
+        if (amount > 0) {
+          // Each quantity = 1% damage boost
```

Comment:
1%? not 100%?

Reply:
Yes! Just note that this comment is referring to modifier quantity and not spell quantity: Each cast of the debilitate spell still increases incoming damage by 100% (but does so by adding 100 stacks of the modifier). Doing it this way just means it's possible to make an enemy take +x% damage where x isn't a multiple of 100. So we could make a rune that applies say, 25% debilitate instead of 100%, if 100% is too large of a number for the conditions of the rune to trigger